packages/rpcs/cfx_signTransaction/index.js

```diff
@@ -131,15 +132,25 @@ export const main = async args => {
 
   let raw
   if (fromAddr.account.accountGroup.vault.type === 'hw') {
-    raw = await signWithHardwareWallet({
-      args,
-      tx: newTx,
-      addressId: fromAddr.eid,
-      device: fromAddr.account.accountGroup.vault.device,
-    })
+    if (dryRun) {
+      raw = cfxSignTransaction(
+        newTx,
+        '0x617a260992af90f20fbbe43cd2e2a4edb9a0ec2ded468de133797184be3cd818',
+        network.netId,
+      )
+    } else {
+      raw = await signWithHardwareWallet({
+        args,
+        tx: newTx,
+        addressId: fromAddr.eid,
+        device: fromAddr.account.accountGroup.vault.device,
+      })
+    }
   } else {
-    const pk = await wallet_getAddressPrivateKey({address: newTx.from})
+    let pk = await wallet_getAddressPrivateKey({address: newTx.from})
 
+    if (dryRun)
+      pk = '0x617a260992af90f20fbbe43cd2e2a4edb9a0ec2ded468de133797184be3cd818'
```

Comment:
this pk is for testing?

Reply:
It's for dry run only. 
Sign everything with this pk when dry run to ensure the tx is singable